include/deal.II/lac/relaxation_block.h

```diff
@@ -147,10 +147,23 @@ class RelaxationBlock :
      * </ol>
      */
     std::vector<std::vector<unsigned int> > order;
+
     /**
      * Return the memory allocated in this object.
      */
     std::size_t memory_consumption() const;
+
+    /**
+     * Temporary ghost vector that is used in the Relaxation method when
+     * performing parallel MPI computations. The user is required to have this
+     * point to an initialized vector that contains at least the locally
+     * active level DoFs.
```

Comment:
Ah, it finally becomes clear to me how this is supposed to work -- this is a `public` variable, rather than something the class uses internally. Have you thought about whether there are alternatives?

I think it would be useful if the documentation of whatever function(s) this is used in also stated that the user needs to set such a pointer. It would also be useful to say whether this vector will be used for reading or writing and, in the former case, what it is supposed to contain.